### ISSUE TITLE
Major bump when introducing a breaking change

### ISSUE THREAD

**Issue by djmadeira** [oblador/angular-scroll#163]
It looks like this commit broke our build process. https://github.com/oblador/angular-scroll/commit/e8291496b4386e060b3e0eac0ececda35a73df37

We aren't yet using angular as a module, so we get "cannot resolve module" errors when compiling with webpack.

We're locking our version to 0.7.3, but can you change the version to 1.0.0 so we can still get bug fixes?

**Comment by oblador** [comment]
You're right, actually didn't think it would break existing installations. 

Can you try using `require('angular-scroll/angular-scroll')` instead?

**Comment by oblador** [comment]
But btw, I'm guessing you use angular via some kind of CDN or wevs. Without much webpack knowledge, shouldn't your setup use something like this? 
```
new webpack.ProvidePlugin({
    "window.angular": "angular"
})
```

**Comment by oblador** [comment]
Looked into webpack docs, there seems to be a way of using CDN with webpack using:
```
{ externals: { angular: "angular" } }
```

**Comment by oblador** [comment]
Deleted 0.7.4 from npm and released [1.0.0](https://github.com/oblador/angular-scroll/releases/tag/v1.0.0) instead.
